superset-frontend/src/profile/components/Favorites.tsx

```diff
@@ -17,12 +17,13 @@
  * under the License.
  */
 import React from 'react';
+import rison from 'rison';
 import moment from 'moment';
 import { t } from '@superset-ui/core';
 
 import TableLoader from '../../components/TableLoader';
 import { Slice } from '../types';
-import { User, Dashboard } from '../../types/bootstrapTypes';
```

Comment:
I think this PR removes the last reference to this type. Think it's fair game to delete it? If not, we should add a deprecation comment on it.